packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3548,8 +3548,8 @@ class PositionedDirectional extends StatelessWidget {
 /// horizontal) or [Column] (if it's vertical) instead, because that will be less
 /// verbose.
 ///
-/// To cause a child to expand to fill the available vertical space, wrap the
-/// child in an [Expanded] widget.
+/// To cause a child to expand to fill the available space on its main axis,
```

Comment:
on its main axis sounds a bit like the child's axis

Maybe:
to fill the available space in the [direction] of this widget's main axis,

Reply:
You're right, "its" was ambiguous. Updated.